core/tst/software/aws/toolkits/core/region/ToolkitRegionProviderTest.kt

```diff
@@ -0,0 +1,32 @@
+// Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package software.aws.toolkits.core.region
+
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.Test
+import software.aws.toolkits.core.utils.test.aString
+
+class ToolkitRegionProviderTest {
```

Comment:
A lot of tests for this are down in AwsRegionProviderTest FYI

Reply:
moved